Makefile

```diff
@@ -94,7 +94,8 @@ tests:
 	go test ./...
 
 vet:
-	go vet ./...
+	go tool vet --all .
+	! go tool vet --shadow . 2>&1 | grep -vE 'declaration of err shadows declaration at (proto/text.go:535|proto/text.go:570|protoc-gen-gogo/main.go:84)'
```

Comment:
Ok well maybe we shouldn't be using a tool for automated checking if it is producing false positives.